app/webpacker/components/RegistrationsV2/RegistrationEdit/RegistrationEditor.jsx

```diff
@@ -200,12 +202,38 @@ export default function RegistrationEditor({ competitor, competitionInfo }) {
         )}
         {registrationEditDeadlinePassed && (
           <Message>
-            The Registration Edit Deadline has passed!
-            {' '}
-            <strong>Changes should only be made in extraordinary circumstances.</strong>
+            {I18n.t('registrations.errors.edit_deadline_passed')}
           </Message>
         )}
-        <Header>{competitor.name}</Header>
+        <Header>
+          {`${competitor.name} `}
+          (
+          {/* The onClick functions prevent Semantic UI from rendering large icons indicating new
+          tabs */}
```

Comment:
This sounds very wrong. I suspect that it has something to do with how you're using `onClick`.